setup.cfg

```diff
@@ -9,9 +9,11 @@ classifiers =
     Operating System :: OS Independent
     Programming Language :: Python
     Programming Language :: Python :: 3.6
+long_description = file: READ.md
+long_description_content_type = text/markdown
 project_urls =
-    Documentation = https://cloudstate.io/docs/user/lang/index.html
-    Source = https://github.com/cloudstateio/python-support
+    Documentation = https://akkaserverless.io/docs/user/lang/index.html
```

Comment:
There might be a reason to mix akkaserverless.io (here) and akkaserverless.com (at url) from time to time; or not.